USER_MANUAL.MD

```diff
@@ -0,0 +1,5 @@
+# SpatialOS GDK for Unreal Test Gyms Project 
+
+#### How to add a gym 
+* Make a copy of the empty gym and give it a descriptive name.
```

Comment:
Do we mean clone the repository? Do we mean download and make a copy in our file system?
How do we make a copy?